shopping-cart/kart.js

```diff
@@ -0,0 +1,17 @@
+import { renderTableRow } from './kart.utilities.js';
+import { cartArray, kittens } from '../products/data.js';
+import { calcTotal } from '../utils.js';
+
+const table = document.querySelector('tbody');
+for (let i = 0; i < cartArray; i++) {
+    const kittens = cartArray[i];
```

Comment:
const kittens = cartArray[i] 
this will be one item from the cartArray, I used to always write:
const item = cartArray[i] to make it a little more clear. Choice is yours but it does make it less confusing.